Week1/homework/app.js

```diff
@@ -1,11 +1,163 @@
 'use strict';
 {
+
+
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    "Anna Frank haar leven",
+    "Meisje in de trein",
+    "Hanna's verjaardag",
+    "Een nieuwe toekomst",
+    "De geur van regen",
+    "Detective Orhan",
+    "Last minute",
+    "De man van de bloemen",
+    "Ren Amina ren",
+    "Het diner"
   ];
+//1st method(for step 2.3)
+  /*function makeList() {
+    let bookcontainer = document.getElementById("mybooks");
+    let ul = document.createElement("ul");
+    bookcontainer.appendChild(ul);
+    for (let i = 0; i<bookTitles.length; i++) {
+      let li = document.createElement("li");
+      let content = document.createTextNode(bookTitles[i]);
+      li.appendChild(content);
+      ul.appendChild(li);
+    }
+    bookcontainer.appendChild(ul);
+  }
+  makeList();*/
+  
+  //2nd method(1(for step 2.3))
+  
+  /*bookTitles.forEach(function(book) {
+    var li = document.createElement("li");
+    var text = document.createTextNode(book);
+    li.appendChild(text);
+    document.getElementById("mybooks").appendChild(li);
+  });*/
+
+//step 2.4))
+ const bookDetails = {
+  "Anna Frank haar leven" : {
+     title : "Anna Frank haar leven",
+     language : "Dutch",
+     Author : "Anna Frank",
+     
+     
+   },
+   "Meisje in de trein" : {
+    title : "Meisje in de trein",
+    language : "Dutch",
+    Author : "Paula Hawkins",
+    
+  }, 
+   "Hanna's verjaardag" : {
+    title : "Hanna's verjaardag",
+    language : "Dutch",
+    Author : "Gull Akerblom",
+   
+   }, 
+   "Een nieuwe toekomst" : {
+    title : "Een nieuwe toekomst",
+    language : "Dutch",
+    Author : "Conor Grennan",
+   
+   },
+   "De geur van regen" : {
+    title :     "De geur van regen",
+    language : "Dutch",
+    Author : "Lieneke Dijkzeul",
+    
+   }, 
+   "Detective Orhan" : {
+    title :     "Detective Orhan",
+    language : "Dutch",
+    Author : "Sadik Yemni",
+    
+   }, 
+   "Last minute" : {
+    title :     "Last minute",
+    language : "Dutch",
+    Author : "Linda Van Rijn",
+    
+   },
+   "De man van de bloemen" : {
+    title :     "De man van de bloemen",
+    language : "Dutch",
+    Author : "Johan Van Caeneghem",
+    
+   }, 
+   "Ren Amina ren": {
+    title :     "Ren Amina ren",
+    language : "Dutch",
+    Author : "Annelie Drewsen",
+   
+   },
+   "Het diner" : {
+    title :     "Het diner",
+    language : "Dutch",
+    Author : "Herman Koch",
+    
+  }
+ }
+ //console.log(bookDetails);
+
+ const bookImg = {
```

Comment:
you have a very nice `base64` images, but you should be aware of legibility, it takes a very long line which means with `wrap` option in a Code editor it's long scrolling, and and object ruiner :boom:, don't do that again :slightly_smiling_face: unless you have the url address for these images.